apps/web/components/CommunityCard.tsx

```diff
@@ -1,236 +1,233 @@
-                    className={`block h-6 w-6 stroke-2 ${open && "rotate-180"}`}
-                    aria-hidden="true"
-                  />
-                </Button>
-              )} */}
-            </div>
-            <h5 className="mt-4 font-bold">
-              Funding pools ( {fundingPools.length} )
-            </h5>
-            <div
-              className={`flex w-full transform flex-wrap gap-4 overflow-x-auto transition-height duration-200 ease-in-out  `}
-            >
-              {fundingPools.map((pool, i) => (
-                <PoolCard tokenGarden={tokenGarden} {...pool} key={i} />
-              ))}
-            </div>
-
-            <h5 className="mt-4 font-bold">
-              Pools in Review ( {poolsInReview.length} )
-            </h5>
-            <div
-              className={`mt-2 flex w-full transform flex-wrap gap-4 overflow-x-auto transition-height duration-200 ease-in-out  `}
-            >
-              {poolsInReview.map((pool, i) => (
-                <PoolCard
-                  tokenGarden={tokenGarden}
-                  {...pool}
-                  key={i}
-                  enabled={false}
-                />
-              ))}
-            </div>
-
-            {}
-
-            {/* IncreasePower funcionality - alpha test */}
-            <h3 className="mt-10">Your stake in the community</h3>
-            <IncreasePower
-              communityAddress={communityAddress as Address}
-              registerToken={registerToken as Address}
-              connectedAccount={accountAddress as Address}
-              tokenSymbol={tokenGarden?.symbol as string}
-              registerTokenDecimals={tokenGarden?.decimals as number}
-              registerStakeAmount={registerStakeAmount as bigint}
-              memberStakedTokens={BigInt(memberStakedTokens)}
-            />
-          </div>
-        </main>
-      </div>
-    </>
-  );
-}
+"use client";
+import { useEffect, useState } from "react";
+import { RegisterMember } from "@/components";
+import {
+  UserGroupIcon,
+  BuildingOffice2Icon,
+} from "@heroicons/react/24/outline";
+import {
+  PoolCard,
+  IncreasePower,
+  CommunityProfile,
+  FormLink,
+} from "@/components";
+import { usePathname } from "next/navigation";
+import { Address, useAccount } from "wagmi";
+import {
+  TokenGarden,
+  getCommunitiesByGardenQuery,
+  isMemberDocument,
+  isMemberQuery,
+} from "#/subgraph/.graphclient";
+import { formatTokenAmount } from "@/utils/numbers";
+import { useUrqlClient } from "@/hooks/useUqrlClient";
+import { getChainIdFromPath } from "@/utils/path";
+import useSubgraphQueryByChain from "@/hooks/useSubgraphQueryByChain";
+
+export type StakesMemberType =
+  isMemberQuery["members"][number]["memberCommunity"];
+
+type CommunityQuery = NonNullable<
+  NonNullable<getCommunitiesByGardenQuery["tokenGarden"]>["communities"]
+>[number];
+
+type CommunityCardProps = CommunityQuery & {
+  tokenGarden: TokenGarden | undefined;

```

Comment:
TokenGarden, same possible issue if using different query on it

Reply:
Probably not my changes

Reply:
I see, but that should be the correct one anyway.

Reply:
Yeah when we will sync with lucho I think this component will change a lot